components/scream/src/control/surface_coupling.cpp

```diff
@@ -12,30 +13,51 @@ SurfaceCoupling (const field_mgr_ptr& field_mgr)
 {
   auto grid = m_field_mgr->get_grid();
 
-  m_num_cols  = grid->get_num_local_dofs();
+  m_num_cols = grid->get_num_local_dofs();
+  m_num_levs = grid->get_num_vertical_levels();
+
+  using namespace ShortFieldTagsNames;
+  using namespace ekat::units;
+
+  FieldLayout layout { {COL}, {m_num_cols} };
+  Units nondim(0,0,0,0,0,0,0);
```

Comment:
Minor (and probably subjective): I think most people will be confused if this ctor is used. You could do something less puzzling by doing
```
auto nondim = Units::nondimensional();
```

Reply:
Ahh didn't realize there was a simpler way. I'll definitely change.